sig-instrumentation/charter.md

```diff
@@ -0,0 +1,145 @@
+# SIG Instrumentation Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README].
+
+## Scope
+
+Covers best practices for cluster observability through metrics, logging, and events across all Kubernetes components and development of relevant components.
+
+### In scope
+
+Coordinating metric requirements of different SIGs for other components through finding common APIs.
+
+### Out of scope
+
+Outline of things that could be confused as falling into this SIG but don't
```

Comment:
Topics we don't cover:
- event (metric, log) processing
- prescribing what are error states

Reply:
I think these are reasonable additions.